jshift-kit-quarkus/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>jshift-kit-build</artifactId>
+        <groupId>io.jshift</groupId>
+        <version>0.1-SNAPSHOT</version>
```

Comment:
Your parent section is missing a relative path to parent pom.xml

Reply:
Fixed

Reply:
I mistakenly put `jshift-kit-build` as the parent instead of `jshift-kit-parent` and it passed on my local machine due to local maven cache/repositories